lib/hammer_cli_foreman/trend.rb

```diff
@@ -0,0 +1,45 @@
+module HammerCLIForeman
+
+  class Trend < HammerCLIForeman::Command
+
+    resource :trends
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _("Id")
+        field :label, _("Name")
+      end
+
+      build_options
+    end
+
+    class CreateCommand < HammerCLIForeman::CreateCommand
+      success_message _("Trend %<trendable_type>s created.")
```

Comment:
If the name is optional, I think it makes sense to use the type, but if we use the type, shouldn't we change the message to `Trend for %<trendable_type> created.`? The same for `DeleteCommand`.